assembly/Cargo.toml

```diff
@@ -35,5 +35,5 @@ thiserror.workspace = true
 [dev-dependencies]
 miden-mast-package = { workspace = true, features = ["arbitrary"] }
 miden-stdlib.workspace = true
-pretty_assertions = "1.4"
+insta = "1.43.1"
```

Comment:
The [quickstart docs](https://insta.rs/docs/quickstart/) recommend compiling insta in release mode for faster test times - probably a good idea to do so? From the docs:

```toml
[profile.dev.package]
insta.opt-level = 3
similar.opt-level = 3
```